configure.ac

```diff
@@ -704,6 +685,25 @@ if test "$with_skey" = "yes"; then
 	]])],[AC_DEFINE(SKEY_BSD_STYLE, 1, [Define to support newer BSD S/Key API])],[])
 fi
 
+dnl Checks for header files.
+AC_CHECK_HEADERS(crypt.h utmp.h \
```

Comment:
not exactly a new issue, but this (and funcs below) should ideally be using the `_ONCE` variant to avoid double-checking for the headers